unit_tests/lib/fake_docker_cluster.py

```
@@ -0,0 +1,89 @@
+
+    Supports:
+    - ``node_setup`` (needs ``params``, ``logdir``, ``test_config``)
+    - ``_create_nodes`` / ``_get_nodes`` (need ``racks_count``, ``nodes``,
+      ``node_prefix``)
+    """
+
+    # noinspection PyMissingConstructor
+    def __init__(
+        self,
+        params: dict,
+        logdir: str = "/tmp",
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)